libpod/container_api.go

```diff
@@ -372,7 +389,6 @@ func (c *Container) Mount(label string) (string, error) {
 	}
 	c.state.Mountpoint = mountPoint
 	c.state.Mounted = true
-	c.config.MountLabel = mountLabel
```

Comment:
Why are you removing this line?

Reply:
Changes to c.config are not persistent, so this will disappear the next time we refresh from the database. Persistent changes to a container's mount label are not supported because of this. If we really need to store the current mount label we're using, we need a new spot for it in database state.

Reply:
We need to make sure that a stopped container, when restarted has the same label, we might be doing this through the use of saving process label.  But bottom line is

podman run ...
podman stop
podman start
Labels must be the same.

Reply:
This only applies to 'podman mount' which can override our SELinux mount label - otherwise we'll always use the label provided to us at creation

Reply:
I noticed it when editing the file and fixed it at the same time

Reply:
Ok podman mount should not have the ability to change this since that could also cause issues.

Reply:
We should remove that, it should get the label from the container.

Reply:
Music to my ears. Mind if I do that in a followon PR?

Reply:
SGTM